website/cue/reference/components/sources/prometheus_remote_write.cue

```diff
@@ -81,6 +81,14 @@ components: sources: prometheus_remote_write: {
 				are emitted as gauges.
 				"""
 		}
+
+		duplicate_tags: {
+			title: "Duplicate Tags"
+			body: """
+				Duplicate tag names are invalid within Prometheus. Prometheus itself will reject a metric with duplicate
+				tags. Vector will accept the metric, but will only take the last tag specified.
```

Comment:
I like this version ( "Duplicate tag names are invalid within Prometheus. Prometheus itself will reject a metric with duplicate tags." )

, better than the one in the other files ( "Duplicate tag name are invalid within Prometheus and Prometheus will reject a metric with duplicate tags." )

It seems like they could all be consistent ?

Reply:
It is debatable, but I think they are different contexts, and thus different wording. 

For the sinks we are saying that if we send this data to Prometheus it will get rejected, so we need to sort it out. Prometheus is directly involved in this situation.

 For the sources we are saying that we accept them, but this alternative program 'Prometheus' will reject them, however this alternative program is not directly involved in the current situation.